### PR TITLE
AAE-11900 RSA and realm oath2 token verification without using Keycloak client

### PR DESCRIPTION
**Pull request by jesty**

### CODE REVIEW

File: activiti-cloud-service-common/activiti-cloud-services-common-identity-keycloak/src/test/java/org/activiti/cloud/services/identity/keycloak/PublicKeyValidationCheckIT.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    ObjectMapper objectMapper;
+
+    @Autowired
+    SecurityContextTokenProvider securityContextTokenProvider;
+
+    @Test
+    void should_tokenBeValid() {
+        Jwt accessToken = Jwt.withTokenValue(securityContextTokenProvider.getCurrentToken().get())
+            .header("a", "b")
+            .claim("a", "b")
```
Comment (scaccia) [inline activiti-cloud-service-common/activiti-cloud-services-common-identity-keycloak/src/test/java/org/activiti/cloud/services/identity/keycloak/PublicKeyValidationCheckIT.java:56]: What are the meaning of this fields values?

Comment (jesty) [reply activiti-cloud-service-common/activiti-cloud-services-common-identity-keycloak/src/test/java/org/activiti/cloud/services/identity/keycloak/PublicKeyValidationCheckIT.java:56]: they are mandatory for the builder, but in the end they aren't used.

**Review by scaccia — APPROVED** [review]
LGTM

**Review by scaccia — APPROVED** [review]
LGTM

**Pull request merged**